jenkins-plugin/src/main/resources/org/jenkinsci/plugins/pipeline/maven/GlobalPipelineMavenConfig/config.jelly

```diff
@@ -30,6 +30,9 @@ THE SOFTWARE.
         <f:entry title="${%DAO}">
             <pre><code>${instance.dao.toPrettyString()}</code></pre>
         </f:entry>
+        <f:entry title="${%No Data Storage}" field="noDataStorage">
+            <f:checkbox title="${%Do not store any data}" />
```

Comment:
Use of the negative sense in labels is confusing to users, and in property names is confusing to code readers. Boolean properties should use positive naming: _do_ store data in H2, _do_ use some database.